src/components/book-card/z-book-card-app/styles.css

```diff
@@ -1,47 +1,65 @@
+:host {
+  display: block;
+  width: inherit;
+  min-width: 300px;
+  height: inherit;
+  min-height: 32px;
+  background-color: var(--color-surface01);
+}
+
 :host > div {
   display: flex;
-  width: 315px;
-  height: 47px;
+  width: 100%;
+  height: 100%;
```

Comment:
In questo modo abbiamo un po' di variabilità nelle altezze, ci va bene lo stesso?
![Screenshot from 2024-12-02 11-35-34](https://github.com/user-attachments/assets/a8ad3b6b-2c88-40ba-a3a4-ef3375f73913)

Reply:
Anche in lunghezza
![Screenshot from 2024-12-02 11-58-18](https://github.com/user-attachments/assets/3cfed735-db6b-4f71-aee1-0978d18da37f)

Reply:
Se si vogliono evitare delle misure fisse, credo sia inevitabile. Quindi andranno necessariamente gestite dall'applicazione a seconda del contesto di utilizzo.